### ISSUE TITLE
I don't understand Oauth here

### ISSUE THREAD

**Issue by alxvallejo** [tamland/python-tidal#80]
If i use `login_oauth_simple`, it just spits out:

> Visit link.tidal.com/ABCDEF to log in, the code will expire in 300 seconds

 So if i copy and paste that in Chrome i can log in successfully to a page that says

> Your device was successfully linked to your account

Ok ... now what?

If i run my script again, and check my login via `check = session.check_login()`, it is false.

So how do i get my session id, etc?

**Comment by manuelcortez** [comment]
You need to retrieve all data from your session object the first time you authorize the account, before the script will terminate. It would be something like this:

```
import tidalapi

session = tidalapi.Session()
# Will run until you visit the printed url and link your account
session.login_oauth_simple()
# After this, save the following data somewhere else, for example in a config file.
# session.session_id
# session.access_token
# session.refresh_token
# session.token_type
```

During next executions of your script, you would "load" the oauth session with the data saved earlier:

```
# This assumes you already have variables named access_token, token_type, refresh_token and session_id
# Which contains the data you saved during the authorization process.
import tidalapi

session = tidalapi.Session()
session.load_oauth_session(session_id, token_type, access_token, refresh_token)
# Run everything else you want here.
```

**Comment by NigelDawson** [comment]
session.login_oauth_simple() is not returning the expected url (not that i'm sure what that is).   A 401 error is thrown with https://auth.tidal.com/v1/oauth2/device_authorization.  Hitting this URL the site returns a 500 error.

**Comment by morguldir** [comment]
@NigelDawson see #95, this has been fixed in the 0.6.10 version

I believe @manuelcortez' response answers this issue, but aditionally make sure that you store the acces_token and refresh_token in a safe place, since it gives full access to your TIDAL account.

**Comment by NigelDawson** [comment]
thanks @morguldir it's working.   I had the 0.6.9 straight from pip.  running 0.6.x from git now.   sorry for any hassle.  thanks again.
